internal/controllers/topology/cluster/desired_state.go

```diff
@@ -512,6 +516,10 @@ func computeMachineDeployment(_ context.Context, s *scope.Scope, desiredControlP
 		return nil, errors.Wrapf(err, "failed to compute version for %s", machineDeploymentTopology.Name)
 	}
 
+	// Compute labels for MachineDeployment and Machines
+	machineLabels := mergeMap(machineDeploymentTopology.Metadata.Labels, machineDeploymentBlueprint.Metadata.Labels)
+	machineAnnotations := mergeMap(machineDeploymentTopology.Metadata.Annotations, machineDeploymentBlueprint.Metadata.Annotations)
```

Comment:
similar comments as above: machineDeploymentLabels/annotations + should we add the cluster and topology-owner annotation?

Reply:
Will rename it too. 

All code, related to cluster and topology-owned annotations is a few lines down, together with selectors: https://github.com/kubernetes-sigs/cluster-api/blob/66aacfba3bf653cf648e0d0ea968d10d9cd751b8/internal/controllers/topology/cluster/desired_state.go#L566-L594